src/scss/components/optional/_ratings.scss

```diff
@@ -29,8 +34,14 @@ $star-spacing           : $star-count;
     }
 
     .c-rating--small {
-        height: $star-size--small;
-        width: ($star-size--small * $star-count) + $star-spacing;
+
+        @include smallStarSizing();
+
+        @media only screen and (-webkit-device-pixel-ratio : 3) {
```

Comment:
You can access this through `@include-media` – [see the docs here](https://include-media.com/documentation/#variable-media-expressions)

Reply:
Think it would be something like:

```
@include media('retina3x') { } 
```